WordPress/gutenberg
# ItemGroup: Update button focus styles to use `:focus-visible`

## What?

A small follow-up to https://github.com/WordPress/gutenberg/pull/51576, to use `:focus-visible` rather than `:focus` on the `ItemGroup` buttons.

## Why?


Addresses this comment: https://github.com/WordPress/gutenberg/pull/51576#discussion_r1236497651

## Testing Instructions





Navigate around the inserter menus using both a mouse and keyboard. The focus state should only apply when using a keyboard.
